### PR TITLE
Add event platform to rfxtrx docs

### PR DESCRIPTION
**Pull request by elupus**
## Proposed change

Add mention of event entities for rfxtrx devices.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/111526
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/rfxtrx.markdown [hunk 1]
```diff
@@ -100,6 +103,10 @@ In the options menu, select *Enable automatic add* to enable automatic addition
 
 The RFXtrx integration supports Siemens/LightwaveRF and Somfy RTS roller shutters that communicate in the frequency range of 433.92 MHz.
 
+#### Events
+
+The RFXtrx integration will expose event entities for remotes controlling lights as well as security devices.
```
Comment (klaasnicolaas) [inline source/_integrations/rfxtrx.markdown:108]:
```suggestion
The RFXtrx integration will expose event entities for remotes controlling lights as well security devices.
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_integrations/rfxtrx.markdown [hunk 2]
```diff
@@ -100,6 +103,10 @@ In the options menu, select *Enable automatic add* to enable automatic addition
 
 The RFXtrx integration supports Siemens/LightwaveRF and Somfy RTS roller shutters that communicate in the frequency range of 433.92 MHz.
 
+#### Events
+
+The RFXtrx integration will expose event entities for remotes controlling lights as well security devices.
```
Comment (klaasnicolaas) [inline source/_integrations/rfxtrx.markdown]:
```suggestion
The RFXtrx integration will expose event entities for remotes controlling lights as well as security devices.
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]
I made a mistake, `as well as` was fine. Sorry!

**Review by klaasnicolaas — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged 👍

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @elupus and @klaasnicolaas :+1:

**Pull request merged**